README.md

```diff
@@ -8,7 +8,7 @@ This repository contains all Eclipse related code for Xtext, including
 
 Checkout and run `mvn clean install`.
 
-Note: The [target platform](releng/org.eclipse.xtext.target/org.eclipse.xtext.target-luna.target) used for the Tycho build loads the required Xtext dependencies ([xtext-lib](https://github.com/eclipse/xtext-lib), [xtext-core](https://github.com/eclipse/xtext-core), [xtext-extras](https://github.com/eclipse/xtext-extras)) from their respective p2 repositories on the [Jenkins server](http://services.typefox.io/open-source/jenkins/).
+Note: The [target platform](releng/org.eclipse.xtext.target/org.eclipse.xtext.target-luna.target) used for the Tycho build loads the required Xtext dependencies ([xtext-lib](https://github.com/eclipse/xtext-lib), [xtext-core](https://github.com/eclipse/xtext-core), [xtext-extras](https://github.com/eclipse/xtext-extras)) from their respective p2 repositories on the [Jenkins server](http://ci.eclipse.org/xtext/).
```

Comment:
If you are on it, please also fix the link to the target platform.

Reply:
For info: it was a test how changes through web interface feel for contributors

Reply:
So is this to be merged or to be discarded?